lib/davinci_pas_test_kit/generated/v2.0.1/server_suite.rb

```diff
@@ -79,6 +80,24 @@ class ServerSuite < Inferno::TestSuite
       #   request.query_parameters['token']
       # end
 
+      group 'Subscription Setup' do
+        description %(
+          The Subscription Setup tests verify that the server supports creation of a rest-hook Subscription. The
+          Subscription instance created in these tests will be used for a notification in the pended workflow tests
+          later.
+        )
+        config inputs: { url: { name: :server_endpoint } }
+        input_order :server_endpoint, :smart_credentials, :access_token, :subscription_resource
+        verifies_requirements 'hl7.fhir.us.davinci-pas_2.0.1@141'
+
+        test from: :subscriptions_r4_server_subscription_conformance
+        test from: :pas_server_subscription_input_conformance
+        test from: :subscriptions_r4_server_notification_delivery
+        test from: :subscriptions_r4_server_creation_response_conformance
+        test from: :subscriptions_r4_server_notification_presence
+        test from: :subscriptions_r4_server_handshake_conformance
```

Comment:
do we need the presence test if we're immediately going to check handshake conformance? I see I did something similar in the Subscription server tests, but I think (a) that was because the real target (the notification) was not the first notification needed (the handshake) - in this case, we're verifying the first handshake notification, and (b) that might have been too cute anyway.  So I think my recommendation is to take out `subscriptions_r4_server_notification_presence`.  Does that argument make sense?

Reply:
Yeah, I had this thought, but your comment on the ticket says to use both tests. I'll remove it and verify that it all still works and output makes sense

Reply:
https://github.com/inferno-framework/davinci-pas-test-kit/pull/31/commits/439df49a8aefb04f04d53630dbc1f54dc05459d3